Lib/test/libregrtest/main.py

```diff
@@ -37,6 +37,11 @@
     TEMPDIR = tempfile.gettempdir()
 TEMPDIR = os.path.abspath(TEMPDIR)
 
+if hasattr(sys, 'addaudithook'):
+    # Add an auditing hook for all tests to ensure PySys_Audit is tested
+    def _test_audit_hook(name, args):
+        pass
+    sys.addaudithook(_test_audit_hook)
```

Comment:
please don't add code at top-level. You can add code run in all tests in libregrtest/setup.py function.